ci-operator/step-registry/redhat-lp-chaos/lp/cnv/vm/check/redhat-lp-chaos-lp-cnv-vm-check-commands.sh

```
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -euxo pipefail; shopt -s inherit_errexit
+
+declare vmList="$(cat "${SHARED_DIR}/target-vm-name.txt")"
+
+: '--- Starting Check the VMs ---'
+: "Namespace: ${VM_NAMESPACE}"
+: '------------------------------'
+
+# Check vms status
+function CheckVmRunningStatus() {
+    : '--- Step 2: Checking All VM Status (VMI Ready Condition) ---'
+    declare -i failedCount=0
+    for vmName in ${vmList}; do
+        : "Checking VM: ${vmName}..."
+        oc -n "${VM_NAMESPACE}" wait "vmi/${vmName}" --for condition=Ready --timeout 0 || ((failedCount++))
```

Wouldn't it be better if we fail fast? Any failure will cause us to fail, right?
So why not just do: `oc -n "${VM_NAMESPACE}" wait "vmi/${vmName}" --for condition=Ready --timeout 0`
As soon as it detect a VM is not ready, it just fail. Is there any reason to check the rest?

---

Yes, if any VM failed, there is an issue. but just keep it as is in case we'd like to know how many VMs failed.